test/benchmark.c

```diff
@@ -3,17 +3,110 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <fcntl.h>
-#include <unistd.h>
+#ifndef _WIN32
+	#include <unistd.h>
+#else
+	#include <io.h>
+#endif
 #include <stdlib.h>
 #include <stdio.h>
 #include <time.h>
+#if defined(_WIN32)
```

Comment:
Here you use `#if defined`, a few lines back you use `#ifndef`. I would choose `#ifdef` here. `#if defined ()` is useful when you want to check if multiple flags are defined/undefined as a boolean condition.